src/main/java/io/personium/core/rs/odata/AbstractODataResource.java

```diff
@@ -214,12 +209,8 @@ private MediaType decideOutputFormatFromHeaderValues(String acceptHeaderValue) {
      * @return String up to semicolon
      */
     private String truncateAfterSemicolon(String source) {
-        String result = source;
-        int index = source.indexOf(";");
-        if (index >= 0) {
-            result = source.substring(0, index);
-        }
-        return result;
+        String[] splited = source.split(";");
```

Comment:
This should be `source.split("[ \t]*;")` or `source.split("[ \t]*;", 2)`.

https://tools.ietf.org/html/rfc7231#section-5.3.2
https://tools.ietf.org/html/rfc7230#section-3.2.3

Reply:
I modified it, too.